celery_with_django/main_app/views.py

```diff
@@ -0,0 +1,14 @@
+from django.http import HttpResponse
+from .tasks import add, sub
+from send_mail_app.tasks import send_mail_func
+
+def test(request):
+    result_add = add.delay(10,20)
```

Comment:
Don't think you need delay here. As you put the task `sleep` call inside the task.

Also Can you document a comment how the `delay` works here?

Reply:
added comments how delay works.